packages/cli-main/src/cli/commands/auth/logout.test.ts

```diff
@@ -0,0 +1,36 @@
+import Logout from './logout'
+import {beforeEach, describe, expect, it, vi} from 'vitest'
+import {session, store} from '@shopify/cli-kit'
+
+beforeEach(() => {
+  vi.mock('@shopify/cli-kit', async () => {
+    const module: any = await vi.importActual('@shopify/cli-kit')
+    return {
+      ...module,
+      session: {
+        ...module.session,
+        logout: vi.fn(),
+      },
+      store: {
+        ...module.store,
+        removeSession: vi.fn(),
+        clearAllAppInfo: vi.fn(),
+      },
+    }
+  })
+})
+
+describe('logs out', () => {
+  it('removes session', async () => {
+    await Logout.run()
+
+    expect(session.logout).toHaveBeenCalledOnce()
+    expect(store.removeSession).toHaveBeenCalledOnce()
+  })
+
+  it('removes all app information', async () => {
+    await Logout.run()
+
+    expect(store.clearAllAppInfo).toHaveBeenCalledOnce()
```

Comment:
We try to add Given-When-Then comments to simplify the structure of the tests

```suggestion
    // When
    await Logout.run()

    // Then
    expect(session.logout).toHaveBeenCalledOnce()
    expect(store.removeSession).toHaveBeenCalledOnce()
  })

  it('removes all app information', async () => {
    // When
    await Logout.run()

    // Then
    expect(store.clearAllAppInfo).toHaveBeenCalledOnce()
```